model/version.go

```diff
@@ -82,6 +82,7 @@ type Version struct {
 	TriggerID    string `bson:"trigger_id,omitempty" json:"trigger_id,omitempty"`
 	TriggerType  string `bson:"trigger_type,omitempty" json:"trigger_type,omitempty"`
 	TriggerEvent string `bson:"trigger_event,omitempty" json:"trigger_event,omitempty"`
+	TriggerSHA   string `bson:"trigger_sha,omitempty" json:"trigger_sha,omitempty"`
```

Comment:
Since this is only used for the case where it's _not_ triggered by an upstream build but rather an untracked commit, I think it'd be helpful to include this.

Also in the other case, is TriggerID used to store the versionID as opposed to the project ID? Might be helpful also to clarify that this field could have one of those two values then. I'm also a little hesitant about this though -- it seems like an ID field should be unique.

Reply:
Added clarifying comments. Can you clarify what you mean by an ID field should be unique? They are all referring to unique IDs of other documents. Do you mean the naming of this field is unideal? We can name it `TriggeringEntity` or something similar

Reply:
Ah i actually misunderstood what triggerID was for, this works for me.